packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -199,36 +196,17 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
   // updates in sync mode.)
   let expirationContext: ExpirationTime = NoWork;
 
-  // Keeps track of whether we're currently in a work loop.
   let isPerformingWork: boolean = false;
 
-  // Keeps track of whether the current deadline has expired.
-  let deadlineHasExpired: boolean = false;
-
-  // Keeps track of whether we should should batch sync updates.
-  let isBatchingUpdates: boolean = false;
-
-  // This is needed for the weird case where the initial mount is synchronous
-  // even inside batchedUpdates :(
-  let isUnbatchingUpdates: boolean = false;
-
   // The next work in progress fiber that we're currently working on.
   let nextUnitOfWork: Fiber | null = null;
+  let nextRoot: FiberRoot | null = null;
```

Comment:
Isn't this more like "currentlyProcessingRoot"? The "next" part is confusing here. Although for most of the life time of `nextUnitOfWork` it is also confusingly named.

Reply:
Yeah we should do a renaming pass. I always assumed you went with “next” to avoid confusion with the current tree.